Library/Formula/swiftgen.rb

```diff
@@ -0,0 +1,41 @@
+class Swiftgen < Formula
+  desc "Collection of Swift tools to generate Swift code"
+  homepage "https://github.com/AliSoftware/SwiftGen"
+  url "https://github.com/AliSoftware/SwiftGen/archive/0.4.4.tar.gz"
+  sha256 "429e50b33d1cd3e34fa81b5becff865ce5663284380e91602783bfa109c96b83"
+  head "https://github.com/AliSoftware/SwiftGen.git"
+
+  depends_on :xcode => "7.0"
+
+  def install
+    # Install all the binaries
+    ENV["SWIFTGEN_VERSION"] = "#{version} (via Homebrew)"
+    rake "install[#{bin},/]"
+
+    # Install the various test fixtures in pkgshare needed for `brew test`
+    pkgshare.install(
+      "Tests/Assets/fixtures/Images.xcassets" => "Images.xcassets",
+      "Tests/Assets/expected/FileDefaults.swift.out" => "assets.swift",
+      "Tests/Colors/fixtures/colors.txt" => "colors.txt",
+      "Tests/Colors/expected/FileDefaults.swift.out" => "colors.swift",
+      "Tests/L10n/fixtures/Localizable.strings" => "Localizable.strings",
+      "Tests/L10n/expected/FileWithDefaults.swift.out" => "l10n.swift",
+      "Tests/Storyboard/fixtures/Message.storyboard" => "Message.storyboard",
+      "Tests/Storyboard/expected/MessageWithDefaults.swift.out" => "storyboard.swift",
+    )
+  end
+
+  test do
+    output = `#{bin}/swiftgen-assets #{pkgshare}/Images.xcassets`.strip
```

Comment:
I have no problem changing it to use `shell_output()`, but fmi what's wrong with backticks?